From bad1d7edba0addd5cb925d237242edbfbbf2f108 Mon Sep 17 00:00:00 2001
From: Yuxi Sun <b36102@freescale.com>
Date: Fri, 2 Dec 2011 11:12:28 +0800
Subject: [PATCH] ENGR00163698 MX53 ARD: fix typo error for pwm1 pad disable function

Fix typo error for pwm1 pad disable function.

Signed-off-by: Yuxi Sun <b36102@freescale.com>
(cherry picked from commit d04b2646528b586baeecc1f128508b5363e7ed63)
---
 arch/arm/mach-mx5/mx53_ard.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/mach-mx5/mx53_ard.c b/arch/arm/mach-mx5/mx53_ard.c
index 6550ca9..378e1c1 100644
--- a/arch/arm/mach-mx5/mx53_ard.c
+++ b/arch/arm/mach-mx5/mx53_ard.c
@@ -393,9 +393,9 @@ static void disable_pwm1_pad(void)
 {
 	mxc_iomux_v3_setup_pad(mx53_ard_pwm_pads[2]);
 
-	gpio_request(ARD_PWM2_OFF, "pwm2-off");
-	gpio_direction_output(ARD_PWM2_OFF, 1);
-	gpio_free(ARD_PWM2_OFF);
+	gpio_request(ARD_PWM1_OFF, "pwm1-off");
+	gpio_direction_output(ARD_PWM1_OFF, 1);
+	gpio_free(ARD_PWM1_OFF);
 }
 
 static struct mxc_pwm_platform_data mxc_pwm1_platform_data = {
-- 
1.5.4.4

